WalletWasabi/Blockchain/Transactions/TransactionSummary.cs

```diff
@@ -10,17 +10,33 @@ namespace WalletWasabi.Blockchain.Transactions;
 public class TransactionSummary
 {
 	public DateTimeOffset DateTime { get; set; }
-	public Height Height { get; set; }
+	public Height Height { get; init; }
 	public Money Amount { get; set; }
 	public LabelsArray Labels { get; set; }
-	public uint256 TransactionId { get; set; }
-	public uint256? BlockHash { get; set; }
-	public int BlockIndex { get; set; }
-	public bool IsOwnCoinjoin { get; set; }
-	public IEnumerable<Output> Outputs { get; set; }
-	public IEnumerable<IInput> Inputs { get; set; }
+	public uint256 TransactionId { get; init; }
+	public uint256? BlockHash { get; init; }
+	public int BlockIndex { get; init; }
+	public bool IsOwnCoinjoin { get; init; }
+	public IEnumerable<Output> Outputs { get; init; }
+	public IEnumerable<IInput> Inputs { get; init; }
 	public Money OutputAmount => Outputs.Sum(x => x.Amount);
 	public Money? InputAmount => Inputs.Any(x => x.Amount == null) ? null : Inputs.Sum(x => x.Amount);
 	public Money? Fee => InputAmount != null ? InputAmount - OutputAmount : null;
-	public IEnumerable<BitcoinAddress> DestinationAddresses { get; set; }
+	public IEnumerable<BitcoinAddress> DestinationAddresses { get; init; }
+	public int VirtualSize { get; init; }
+	
+	public FeeRate? FeeRate
+	{
+		get
+		{
+			if (Fee is null)
+			{
+				return null;
+			}
+
+			var rate = (double) Fee.Satoshi / VirtualSize;
+			var money = new Money((decimal)rate, MoneyUnit.Satoshi);
+			return new FeeRate(money * 100);
```

Comment:
Turbolay mentioned a multiplier of 1000 not 100, but why not just do:

```suggestion
			return new FeeRate(Fee, VirtualSize);
```

Note: I tested both and none of these options solve the difference of confirmation time between Preview Transaction and Transaction details. (Mentioned here https://github.com/zkSNACKs/WalletWasabi/pull/11092#pullrequestreview-1542483888)